stylesheets/styles.css

```diff
@@ -0,0 +1,137 @@
+* {
+  font-family: "Helvetica Neue";
+}
+
+/*****Navigation bar at top of page*****/
+header {
+  border: 1px solid;
+  width: 100%;
+  top: 0px;
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+}
+
+nav,
+h1 {
+  margin: 0.67em;
+
+}
+
+nav {
+  margin-top: 1em;
+}
+
+.nav-bar{
+  padding: 0;
+}
+
+.nav-link{
+  list-style-type: none;
+  display: inline;
+  padding: 10px;
+  color: black;
+}
+.nav-link a{
+  color: black;
+  text-decoration: none;
+}
+
+.nav-link a:hover{
+  color: pink;
+}
+
+.home-button {
+  color: black;
+  text-decoration: none;
+}
+
+.home-button:hover {
+  color: pink;
+}
+
+/*******Body*******/
+
+body {
+  padding-top: 100px;
+  margin: 0px;
+}
+/*****Section******/
+
+.main-section {
+  border-bottom: 1px solid;
+  width: 100%;
+  height: 300px;
+  padding-bottom: 110px;
+}
+
+.main-section h2 {
+  width: 20%;
+  /*background-color: red;*/
+  margin-left: 5%;
+  /*border: solid;*/
+  font-size: 4rem;
+  text-align: center;
+  margin-top: 130px;
+}
+
+section div {
+  width: 50%;
+  /*background-color: yellow;*/
+  margin-right: 5%;
+  /*border: solid;*/
+  margin-left: 17%;
+  /*float: right;
+  clear: both;*/
+}
+
+.blog {
+  width: 50%;
+  /*background-color: yellow;*/
+  /*margin-right: 5%;*/
+  /*border: solid;*/
+  /*margin-left: 17%;*/
+  float: right;
+  clear: both;
+}
+
+section h2,
+section div{
+  display: inline-block;
+}
+
+.project-images {
+  margin-top: 60px;
+}
+
+.project-images img {
+  border-radius: 100%;
+  padding: 5px;
+}
+
+/******Footer******/
+.footer {
+  position: fixed;
+  bottom: 10px;
+  width: 100%;
+  display: block;
+}
+
+.home-page {
+  font-size: 40px;
+  font-family: "Arial Narrow", Arial, sans-serif;
+  text-align: center;
+  padding-top: 100px;
```

Comment:
Could you have written this(paddings)  in one line instead?  (https://www.w3schools.com/css/tryit.asp?filename=trycss_padding_shorthand)